### ISSUE TITLE
Allow lines longer than 100 symbols if it contains a URI

### ISSUE THREAD

**Issue by Ilya-Gh** [shyiko/ktlint#112]
As stated per [Android Kotlin Style guide](https://android.github.io/kotlin-guides/style.html#line-wrapping) one of the limit exceptions are cases when:

> Lines where obeying the column limit is not possible (for example, a long URL in KDoc).

I would suggest adding this exception to the `max-line-length` rule to allow the following cases:

```
/**
 * "https://www.google.com/search?q=ktlint&rlz=1C5CHFA_enMD736MD737&oq=ktlint+&aqs=chrome..69i57j69i60l4j69i59.1286j0j4&sourceid=chrome&ie=UTF-8"
 */
```

```
loadUrl("https://www.google.com/search?q=ktlint&rlz=1C5CHFA_enMD736MD737&oq=ktlint+&aqs=chrome..69i57j69i60l4j69i59.1286j0j4&sourceid=chrome&ie=UTF-8")
```

I am willing to submit a PR to add the exception if it will be approved.

**Comment by shyiko** [comment]
Hi @Ilya-Gh. 

Thank you for the ticket and PR offer. 

max-line-length is now suppressed in the context of KDoc. 

I'm not sure about second example, though. Considering that this a regular code I feel like "where obeying the column limit is not possible" rule no longer applies here, as it can be rewritten to something like:

```kotlin
loadUrl("https://www.google.com/search" +
    "?q=ktlint" +
    "&rlz=1C5CHFA_enMD736MD737" +
    "&oq=ktlint+" +
    "&aqs=chrome..69i57j69i60l4j69i59.1286j0j4" +
    "&sourceid=chrome" +
    "&ie=UTF-8")
```

**Comment by shyiko** [comment]
Alright, KDoc fix included in [ktlint@0.13.0](https://github.com/shyiko/ktlint/releases/tag/0.13.0). Let me know if you disagree with loadUrl... part.
